### ISSUE TITLE
Options binding does not trigger update event properly when options text (not value) changes

### ISSUE THREAD

**Issue by softwaremills** [knockout/knockout#1419]
Hi! I really love Knockout. Thanks so much for your hard work on it.

My problem is, I have a page that integrates Knockout with Select2. In Knockout 2.x, this worked fine, probably because all bindings would evaluate together, so when the associated select box was updated, the Select2 binding would fire automatically. This is no longer the case in Knockout 3.x, so I've had to be a bit more tricky about things.

In this case, if I have a constant options value, but the options text changes, inserting an observer to ko.bindingHandlers["options"]["update"] does not seem to trigger a change in the element (even though the backing select tag does change).

I've made a jsfiddle page that demonstrates this, both where the options value changes with the text and where it does not.

http://jsfiddle.net/mrmills/MfttX/11/

**Comment by SteveSanderson** [comment]
Hey, thanks for reporting this, and sorry it took essentially forever to respond. This design change (from 2.x to 3.x) was a very deliberate choice we made and we made pretty strenuous efforts to communicate this to KO users and describe ways of adapting applications to fit. In this specific case with Select2, I'm sure there are ways of continuing to integrate with it, but given how long it's been since you posted I suspect you've found a solution by now.

I'll close this since it's by design and unlikely still to be a live issue for you.
